code/modules/clothing/glasses/glasses.dm

```diff
@@ -7,14 +7,14 @@
 	species_fit = list(VOX_SHAPED, GREY_SHAPED)
 
 /obj/item/clothing/glasses/science
-	name = "Science Goggles"
+	name = "science goggles"
 	desc = "nothing"
 	icon_state = "purple"
 	item_state = "glasses"
 	origin_tech = Tc_MATERIALS + "=1"
```

Comment:
Well considering that it is in science round start would mean you could deconstruct a pair to get level two, then be able to make pairs